packages/lib/pod/test/podCrypto.spec.ts

```diff
@@ -56,6 +59,18 @@ describe("podCrypto hashes should work", async function () {
     }
   });
 
+  it("podEdDSAPublicKeyHash should produce unique repeatable results", function () {
```

Comment:
I'd suggest some tests which confirm that the hash of a EdDSA pubkey isn't just a string a hash, and instead has something to do with the EC point of the key.
A white-box approach to that would be to just check against the specific hash code by duplicating it here.
A fuzzier black-box approach might be to hash the same input as a string and confirm the results are different.